packages/packages/react-native-sdk/src/index.tsx

```diff
@@ -33,7 +33,22 @@ export const MetriportWidget = (props: MetriportWidgetProps & WebViewProps) => {
     return <NativeWidget {...props} />;
   }
 
-  const url = props.url ? props.url : "https://connect.metriport.com";
+  let url = props.url ? props.url : "https://connect.metriport.com";
+
+  url = props.sandbox ? `${url}&sandbox=true` : url;
```

Comment:
I think we'll need to change this so the first param is `https://connect.metriport.com?param=blah`, and every subsequent one has the `&`